sentry_sdk/_types.py

```diff
@@ -63,7 +64,7 @@
         "MeasurementValue",
         {
             "value": float,
-            "unit": Optional[MeasurementUnit],
+            "unit": NotRequired[Optional[MeasurementUnit]],
```

Comment:
Are we sure it should still be `Optional`?

Reply:
I would say yes. Gives us the most flexibility.